### DISCUSSION TITLE
Building the server with llava/mmproj/multimodal

### DISCUSSION THREAD

**Discussion by KerseyFabrications** [ggml-org/llama.cpp#5945] [Q&A] [answered]
I'm trying to get the `server` binary working with multimodal but mine is not being built with the `--mmproj` option from the `master` branch. `llava-cli` is being built. Can you tell me if I need to pull a separate branch or need to add any options to get the server working? Thanks!

**Comment by phymbert** [comment, marked as answer]
server multimodal support has been temporarily dropped in #5882

**Reply by KerseyFabrications** [reply]
Cool! Thanks for the reply! I'll time travel a bit with git to try it out.

**Comment by gnusupport** [comment]
Is there multimodal feature from endpoint that I can upload picture through endpoint?
